### DISCUSSION TITLE
Apologies for bulk geoCode request!

### DISCUSSION THREAD

**Discussion by jamesrothwellhughes** [osm-search/Nominatim#2925] [Q&A]
Hi, I recently thoughtlessly wrote a script that used your geocode function (which I normally just use for a user interface based occasional search) in a bulk routine and now receive CONNECTION_REFUSED - not unexpectedly!

Clearly I have disposed of that script now.  Will I be allowed to use the service again as long as it is within the fair usage policy?  If so (I really hope so!), how long do I need to wait?

Many thanks for any answers.

Regards,
James.

**Comment by mtmail** [comment]
Such a block is automatic. Wait 24h and it should work again.
